src/transformers/models/clip/image_processing_clip.py

```diff
@@ -84,6 +84,8 @@ class CLIPImageProcessor(BaseImageProcessor):
             Can be overridden by the `image_std` parameter in the `preprocess` method.
         do_convert_rgb (`bool`, *optional*, defaults to `True`):
             Whether to convert the image to RGB.
+        default_to_square_for_size (`bool`, *optional*, defaults to `False`):
+            The value to be passed to `get_size_dict` as `default_to_square` when computing the image size.
```

Comment:
let's say what it changes depending on this values!

Reply:
```
        use_square_size (`bool`, *optional*, defaults to `False`):
            The value to be passed to `get_size_dict` as `default_to_square` when computing the image size. If the
            `size` argument in `get_size_dict` is an `int`, it determines whether to default to a square image or not.
            Note that this attribute is not used in computing `crop_size` via calling `get_size_dict`.
```